src/components/markdown-renderer/replace-components/mermaid/mermaid-chart.tsx

```diff
@@ -36,7 +36,7 @@ export const MermaidChart: React.FC<MermaidChartProps> = ({ code }) => {
 
   const showError = useCallback((error: string) => {
     setError(error)
-    console.error(error)
+    //console.error(error)
```

Comment:
Should this be removed? Or did you comment it out for a test?

Reply:
forgot to readd it. fixed